dev/build_release.py

```diff
@@ -648,6 +647,11 @@ def init_argument_parser(cls, parser):
       parser.add_argument(
           '--do_jar_build', type=bool, default=True,
           help='Build & publish jars independently to GCS.')
+      parser.add_argument('--maven_custom_init_file', default='spinnaker/dev/maven-init.gradle',
```

Comment:
The default here is assuming that you are running from build (or the parent). Is this really the case? I think it would be safer to say
   os.path(os.path.dirname(__file__), 'maven-init.gradle')

Reply:
Done.